chapter_optimization/adagrad_vn.md

```diff
@@ -160,10 +160,10 @@ Alas, this is a rather impractical suggestion.
 Computing eigenvalues and eigenvectors is in general *much more* expensive than solving the actual problem.
 -->
 
-Nếu hệ số điều kiện $\kappa$ lớn, việc giải bài toán tối ưu trở nên khá khó khăn.
+Nếu hệ số điều kiện $\kappa$ lớn, việc giải bài toán tối ưu một cách chính xác trở nên khá khó khăn.
 Ta cần đảm bảo việc lựa chọn phù hợp một khoảng động lớn các giá trị.
-Quá trình phân tích dẫn đến một câu hỏi hiển nhiên dù có phần ngây thơ: liệu ta có thể "cố định" bài toán bằng cách biến đổi không gian sao cho tất cả các trị riêng có giá trị bằng $1$.
-Biến đổi này khá đơn giản theo lý thuyết: ta chỉ cần tính các trị riêng và các vector riêng của $\mathbf{Q}$ nhằm chuyển đổi giá trị của bài toán 
+Quá trình phân tích dẫn đến một câu hỏi hiển nhiên dù có phần ngây thơ rằng: liệu ta có thể "cố định" bài toán bằng cách biến đổi không gian sao cho tất cả các trị riêng đều có giá trị bằng $1$.
+Điều này khá đơn giản trên lý thuyết: ta chỉ cần tính các trị riêng và các vector riêng của $\mathbf{Q}$ nhằm tái tỉ lệ bài toán 
```

Comment:
Từ `rescale` ở đây mang nghĩ chỉnh sửa chung chung `(change)` nhiều hơn là chỉnh sửa về tỉ lệ
```suggestion
Điều này khá đơn giản trên lý thuyết: ta chỉ cần tính các trị riêng và các vector riêng của $\mathbf{Q}$ nhằm biến đổi bài toán 
```

Reply:
Nếu vậy khá vô lý vì câu trên đang mong muốn làm cho tất cả các trị riêng đều có giá trị bằng một, vậy từ `rescale` ở đây em hiểu giống như là scale lại các giá trị của trị riêng

Reply:
`Quá trình phân tích dẫn đến một câu hỏi hiển nhiên dù có phần ngây thơ rằng: liệu ta có thể "cố định" bài toán bằng cách biến đổi không gian sao cho tất cả các trị riêng đều có giá trị bằng $1$. `

Ở đây đúng là em `tái tỉ lệ` trị riêng và vector riêng, nhưng mục địch cuối cùng là để `cố định` hay là biến đổi bài toán để nó cố định

Reply:
À, okie. Em thấy biến đổi mang đủ nghĩa hơn trong câu trên